llumnix/backends/vllm/migrate_backend.py

```diff
@@ -85,48 +86,41 @@ def destory_col(self) -> None:
         logger.info("destory rpc migrate backend success.")
 
     def warmup(self) -> None:
-        self_handle = self.worker_handle_list[self.worker_rank]
-        self.actor.exec_method.remote(self.is_driver_worker, self_handle, "do_send", None, [0])
+        self.actor.exec_method.remote(self.is_driver_worker, "do_send", [0])
 
     def migrate_cache(self, src_handle, src_blocks: List[int], dst_blocks: List[int]) -> None:
         tot_blocks = len(src_blocks)
         rpc_numpy_cache = None
         for start_idx in range(0, tot_blocks, self.num_migration_cache_blocks):
             offset = min(self.num_migration_cache_blocks, tot_blocks - start_idx)
             send_blocks = src_blocks[start_idx:start_idx+offset]
-            ray_obj = self.actor.exec_method.remote(self.is_driver_worker, src_handle, "do_send", None, send_blocks)
+            ray_obj = self.actor.exec_method.remote(self.is_driver_worker, src_handle, "do_send", send_blocks)
             if rpc_numpy_cache is not None:
                 self.do_recv(rpc_numpy_cache, recv_blocks)
             rpc_numpy_cache = ray.get(ray_obj)
             recv_blocks = dst_blocks[start_idx:start_idx+offset]
         self.do_recv(rpc_numpy_cache, recv_blocks)
 
-    def do_send(self, dst_handle, blocks: List[int]):
+    def do_send(self, blocks: List[int]):
```

Comment:
Although dst_handle is not used, retain it to maintain compatibility with the MigrateBackendBase interface.

Reply:
In `RPCMigrateBackend`, `def do_recv(self, rpc_numpy_cache, blocks: List[int])` doesn't follow definition in `MigrateBackendBase`,  It's not a clear api, we only use `src_handle` and `dst_handle` in `RayMigrateBackend`.

Reply:
how about  revising the interface in MigrateBackendBase ?